spec/rspec/support/ruby_features_spec.rb

```diff
@@ -81,6 +81,12 @@ module Support
           expect(RubyFeatures.caller_locations_supported?).to eq(RUBY_VERSION >= '2.0.0')
         end
       end
+
+      specify "#ripper_supported? exists" do
+        RubyFeatures.ripper_supported?
+        # Ensure .ripper_supported? does not load Ripper.
+        expect { ::Ripper }.to raise_error(NameError)
```

Comment:
Given that we now use ripper as part of the failure output, this expectation seems brittle and likely to fail if any earlier specs have failed and you're running the suite with a formatter that prints failures as they happen.  Instead, WDYT of this?

``` ruby
expect {
  RubyFeatures.ripper_supported?
}.not_to change { defined?(Ripper) }
```

The important thing is that `ripper_supported?` does not load it -- but if something else has already loaded it, that's not our concern here.